publication-rest/src/main/java/no/unit/nva/publication/update/UpdatePublicationHandler.java

```diff
@@ -305,9 +313,22 @@ private Publication updateMetadata(UpdatePublicationRequest input,
         new PublishingRequestResolver(resourceService, ticketService, userInstance, customer)
             .resolve(existingPublication, publicationUpdate);
 
+        if (resourceService.shouldUseNewFiles()) {
+            Resource.fromPublication(publicationUpdate).getAssociatedArtifacts().stream()
+                .filter(File.class::isInstance)
+                .map(File.class::cast)
+                .forEach(file -> updateFile(existingPublication, file));
```

Comment:
Why do you update files here? My assumption was that only publication status would change here? (and that files should keep its statuses on unpublish republish

Reply:
Frontend wants to update files metadata using existing endpoint for updating file, so they do not have to implement massive logic change when taking in use new files.

Reply:
ah maybe I just looked at the wrong place, I though this was part of unpublish/republish. my bad! all good then :-)